hugegraph-core/src/main/java/com/baidu/hugegraph/type/define/SchemaStatus.java

```diff
@@ -27,11 +27,13 @@
 
     REBUILDING(3, "rebuilding"),
 
-    DELETING(4, "deleting"),
+    CLEARING(4, "clearing"),
 
-    UNDELETED(5, "undeleted"),
+    DELETING(5, "deleting"),
 
-    INVALID(6, "invalid");
+    UNDELETED(6, "undeleted"),
+
+    INVALID(7, "invalid");
```

Comment:
keep the original code to keep compatible

Reply:
done